tests/sentry/api/endpoints/test_integrations_slack_link_identity.py

```diff
@@ -0,0 +1,279 @@
+from sentry.models import NotificationSetting, UserEmail, UserOption
```

Comment:
I'm a bit confused why any of this file is in this PR, is it related to slash commands?